### DISCUSSION TITLE
CMS Online Validator Tool: "code|1" value required?

### DISCUSSION THREAD

**Discussion by echoi-gwh** [CMSgov/hospital-price-transparency#151] [Q&A] [answered]
According to the documentation and examples, "code|1" is not required but "description" is. However, the online validator seems to think "code|1" is required. Which is true?

**Comment by RidenourK** [comment, marked as answer]
@echoi-gwh. Thank you for your question. 

Please refer to these conditional requirements on the [CMS Hospital Price Transparency Data Dictionary GitHub](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation) site which indicate “If a standard charge is encoded, there must be a corresponding code and code type pairing. The code and code type pairing do not need to be in the first code and code type columns (i.e., code|1 and code|1|type)” and “If a value is encoded in the "code", a value must also be encoded in the corresponding "code type". Conversely, if a value is encoded in the "code type", a value must be encoded in the corresponding "code" (e.g., if a value is encoded in code|2|type, then a value must be encoded in code|2).”

**Reply by echoi-gwh** [reply]
Thank you for this info! I did miss some of these stipulations.

However, even with this new info, I still don't think the validator is checking correctly (or the error message is incorrect).

*Context: Yes, these rows have standard charges.*

1) The rows getting an error have "code|2" filled out. (Side note, "code|2|type" is "LOCAL" if that matters. Only "code|1|type" is "CPT").
2) The error message says specifically that "code|1" needs a value.

**Reply by RidenourK** [reply]
@echoi-gwh. Thank you for your question.

Would you be willing to provide a screenshot of the error you are receiving and a sample of the MRF that is triggering this error to the CMS HPT mailbox at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov) so we may investigate this further?

**Reply by echoi-gwh** [reply]
OH wait, no, I apologize. I just realized MY code does insert "code type" into [code|1|type] regardless of [code|1] being null. That is my bad, the online validator worked once I fixed that. Thank you for getting back to me on my error!
